cdap-test/src/main/java/co/cask/cdap/test/StreamManager.java

```diff
@@ -108,10 +108,10 @@
   /**
    * Get events from the specified stream in the specified interval
    *
-   * @param startTime the start time
-   * @param endTime the end time
+   * @param startTime the start time in seconds or "now-xs" format
+   * @param endTime the end time in seconds or "now-xs" format
    * @param limit the maximum number of events to return
    * @return a list of stream events in the given time range
    */
-  List<StreamEvent> getEvents(long startTime, long endTime, int limit) throws IOException;
+  List<StreamEvent> getEvents(String startTime, String endTime, int limit) throws IOException;
```

Comment:
This is an API change. We should probably deprecate the existing method for a couple of releases, not remove it instantly.